e2e/terraform/scripts/cleanup.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+
+set -e
+
+LOCAL_NAME=$1
```

Comment:
This script is pretty destructive, so we should probably enforce that it's being passed with this argument. Some of these functions would result in destroying _all_ instances of the component, because they have "contains" filters.

I'd also love to get a "I'm gonna destroy _X_, are you sure?" confirmation too, but that's maybe paranoid. 😀